ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.13__jenkins-hypershift4.13.yaml

```diff
@@ -0,0 +1,71 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.13
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
+    COPY . .
+    RUN chgrp -R 0 /go && \
+    chmod -R g=u /go
+    RUN mkdir -p /.cache
+    RUN chgrp -R 0 /.cache && \
+    chmod -R g=u /.cache
+  to: jenkins-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-jenkins-lp-rosa-hypershift
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "osd-*", "failure_type": "all", "classification": "User Setup", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-*", "failure_type": "all", "classification":  "Cluster Creation", "jira_project": "LPTOCPCI"},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "cucushift-hypershift-*", "failure_type": "all", "classification": "Cluster Health Check", "jira_project": "LPTOCPCI"},
+          {"step": "jenkins-smoke-tests", "failure_type": "pod_failure", "classification": "Test Execution", "jira_project": "LPTOCPCI"},
+          {"step": "jenkins-smoke-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "OCPBUGS", "jira_component": "Jenkins"}
+           ]
```

Comment:
Not sure if it will matter, but the indentation here doesn't match the opening bracket.